master/usage/configuration/node.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Configuring a node
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing, and the BGP AS Number (which generally only needs to 
+be changed if you are explicitly configuring BGP peerings with your Calico nodes).
+
+To configure the routing parameters on your node, you have the options of:
+-  Set appropriate flags on the `calicoctl node run` command
+-  Set appropriate environments for the `calico/node` container
+-  Configure the node resource using `calicoctl`
+
+> Note that for nodes that were configured prior to v2.1, the subnet information 
+> configured on those nodes will be set to /32 for IPv4 or /128 for IPv6.  If you are
+> planning on using the `cross-subnet IP-in-IP` feature then you must update your node
+> configuration to have the correct subnet information for each node.
```

Comment:
I had thought about that but it gets tricky...

If the startup processing uses the consumer facing API (which it does) then no it's not possible.  By the time the subnet propagates up to the API it has a subnet associated with it (albeit a /32 or /128).  It is quite possible that (for whatever reason ... e.g. disabling IPIP for a node) that you want to explicitly set the subnet to /32.... so we can't just change the subnet.

If, on the other hand, we hook into the underlying etcd data model then we could see that it has no subnet information and then try to learn it.  But I was a little loathe to do that.